docs/babel.config.js

```diff
@@ -39,7 +39,13 @@ const { version: transformRuntimeVersion } = fse.readJSONSync(
 module.exports = {
   presets: [
     // backport of https://github.com/zeit/next.js/pull/9511
-    ['next/babel', { 'transform-runtime': { corejs: 2, version: transformRuntimeVersion } }],
+    [
+      'next/babel',
+      {
+        'preset-react': { runtime: 'automatic' },
```

Comment:
`next` only uses `automatic` in `react@^17.0.0-rc.1` but disregards the new JSX runtime in React 16 and 15 as well as the other release channels.

We're forcing it in our root config as well so you can look at this as a consistency change as well.